inc/saivlan.h

```diff
@@ -160,13 +160,13 @@ typedef enum _sai_vlan_member_attr_t
     SAI_VLAN_MEMBER_ATTR_VLAN_ID = SAI_VLAN_MEMBER_ATTR_START,
 
     /**
-     * @brief Logical port ID
+     * @brief Bridge port ID. Valid only for .1Q Bridge ports
      *
      * @type sai_object_id_t
-     * @objects SAI_OBJECT_TYPE_PORT
+     * @objects SAI_OBJECT_TYPE_BRIDGE_PORT
```

Comment:
This looks circular. For sub-ports don't you need to know the VLAN id from the packet to look up the bridge port? Shouldn't the PVID and/or VLAN filtering go with the front panel port based on the packet headers? What purpose does it serve to associate the VLAN id with a bridge port like this?